### PR TITLE
Bug/937 mpi extension

### PR DESCRIPTION
**Pull request by kjezek**
Removed all parts related to MPI

### CODE REVIEW

File: src/Synthesiser.cpp [hunk 1]
```diff
@@ -2339,17 +2237,6 @@ void Synthesiser::generateCode(std::ostream& os, const std::string& id, bool& wi
...
-        os << "\n#ifdef USE_MPI\n";
-        os << "souffle::mpi::init(argc, argv);";
-        os << "int rank = souffle::mpi::commRank();";
-        os << "int stratum = (rank == 0) ? " << std::numeric_limits<int>::max() << " : rank - 1;";
-        os << "obj.runAll(opt.getInputFileDir(), opt.getOutputFileDir(), stratum);\n";
-        os << "souffle::mpi::finalize();";
-        os << "\n#endif\n";
-    } else
-#endif
     {
```
Comment (mmcgr) [inline src/Synthesiser.cpp]: These braces are no longer required. They were only needed to enable the, now removed, if else stucture.

File: src/Makefile.am [hunk 2]
```diff
@@ -56,9 +56,6 @@ if SQLITE
 sqlite_sources = ReadStreamSQLite.h WriteStreamSQLite.h
 endif
 
-if MPI
```
Comment (mmcgr) [inline src/Makefile.am]: Also remove the mpi test

**Comment by mmcgr** [comment]
You will also need to modify .travis.yml to remove the mpi tests on travis CI

File: src/Makefile.am (hunk 2 above)
Comment (b-scholz) [reply src/Makefile.am]: Also please reformat the source code in the src directory:
```
clang-format-7 -i *.cpp *.h
```

**Comment by mmcgr** [comment]
Jenkins, test this please

**Comment by mmcgr** [comment]
Jenkins, test this please

**Comment by b-scholz** [comment]
Issue #937

**Pull request merged**